test/integration/admin_tenants_test.rb

```diff
@@ -0,0 +1,123 @@
+require "test_helper"
+
+class AdminUserTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test "an admin sees a tenants link on the dashboard" do
+    admin = create(:admin)
+    ApplicationController.any_instance.stubs(:current_user).returns(admin)
+
+    visit admin_dashboard_path
+
+    assert page.has_content?("Tenants")
```

Comment:
I am going to leave this for now, and we can address it later as it is trivial to change and requires some discussion.